e2e/cypress/integration/archived_channel/join_archived_channel_spec.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import {getAdminAccount} from '../../support/env';
+
+describe('Archived channels', () => {
+    let testTeam;
+    let testUser;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+        });
+    });
+
+    it('MM-T1682 joining an archived public channel via a permalink', () => {
```

Comment:
Suggested change:
`MM-T1682 Join an archived public channel by selecting a permalink to one of its posts`